translationRecorder/app/src/main/AndroidManifest.xml

```diff
@@ -96,6 +96,15 @@
         <service android:name="org.wycliffeassociates.translationrecorder.Recording.WavRecorder" />
         <service android:name="org.wycliffeassociates.translationrecorder.Recording.WavFileWriter" />
 
+        <provider
```

Comment:
what is this needed for?

Reply:
@sarabiaj Since Android api 24, in order for the app to give an access to it's content to other apps, this FileProvider is needed. You can try to share a project on devices with Android 7 and up, it will crash. 

https://developer.android.com/training/secure-file-sharing/setup-sharing